descope/internal/mgmt/audit.go

```diff
@@ -37,6 +37,22 @@ func (a *audit) Search(ctx context.Context, options *descope.AuditSearchOptions)
 	return unmarshalAuditRecords(res)
 }
 
+func (a *audit) CreateEvent(ctx context.Context, options *descope.AuditCreateOptions) error {
+	body := map[string]any{
```

Comment:
I think we should validate that the mandatory fields are populated like we do in the rest of the SDK

Reply:
We don't do that for the search above

Reply:
fixed